### PR TITLE
IDV document expired scenario

### PR DESCRIPTION
**Pull request by ivo-deriv**
Adds test spec for the IDV expired document scenario.

### CODE REVIEW

File: cypress/e2e/wip/KYC/idvExpired.cy.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  beforeEach(() => {
+    cy.c_login()
+    cy.c_navigateToPOIResponsive('Kenya')
+  })
+
+  it('should return document expired', () => {
+    cy.get('select[name="document_type"]').select('Alien Card')
+    cy.get('input[name="document_number"]').type('000000')
+    cy.get('input[name="first_name"]').type('Joe Doe')
+    cy.get('input[name="last_name"]').type('Leo')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/KYC/idvExpired.cy.js:13]: Please use Testing Library locators for this 
can take a look into cypress/e2e/wip/KYC/idvDobMismatch.cy.js

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/KYC/idvExpired.cy.js (hunk 1 above)
Comment (ivo-deriv) [reply cypress/e2e/wip/KYC/idvExpired.cy.js]: have added locators

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**